python/ingestion/standardized_columns.py

```diff
@@ -196,9 +190,16 @@ class Race(Enum):
     # so we do not know the relationship between Hispanic/Latino and race.
     # ETHNICITY_UNKNOWN refers to data that does not know whether the person is
     # Hispanic or Latino. (Some datasets use "ethnicity" to refer to whether
-    # someone is Hispanic or Latino)
+    # someone is Hispanic or Latino).
+    # PrEP data only collects data for the following race groups: Black, White,
+    # Hispanic, and Other. The dataset does nto distinguish what constitutes as other,
+    # the other demographic does not include individuals who are hispanic.
+    # Using the a unknown race and ethnicity instead of other which is not could be anyone
+    # oustide the standard race group is preferred.
```

Comment:
typo `oustide` and in general i think we can make this comment more clear (especially once we decide what is happening with the race group and what we want to call it)